tox.ini

```diff
@@ -10,6 +10,7 @@ usedevelop=True
 deps =
     pytest
     coverage
+    hypothesis
```

Comment:
I think this one should do in the `tests_require` list that you can find in the `setup.py` file: https://github.com/collective/icalendar/blob/68dc8b18caf0806d2fa1a8e0f06027329ad96c4c/setup.py#L20

Then the `requirements_docs.txt` file should install `. [test]` (untested)